start

```diff
@@ -175,9 +175,9 @@ do
 		ELECTRUM_TXS_LIMIT=9000
 	fi
 
-	if [ ! -e "${POPULAR_SCRIPTS_FILE}" ];then
-		generate_popular_scripts
-	fi
+#	if [ ! -e "${POPULAR_SCRIPTS_FILE}" ];then
```

Comment:
I think you commented out the wrong thing.

You left the "add crontab" if block

But commented out the "perform popular_scripts job" if block

If you run this, it will add the cron job but not run the actual job when the file is missing, and the electrs run command will fail when it tries to run the cache operations on a file that doesn't exist.

Reply:
Also, switching the if block you comment out will change the behavior from

"Always run the popular_script job before running electrs" (current)

to

"Only run the popular_script job when the file doesn't exist. Otherwise, the cron job should have kept the file up to date for us so we shouldn't need to run it" (This PR)